<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CommentBrowserWindow</class>
 <widget class="QDialog" name="CommentBrowserWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>774</width>
    <height>527</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="5,0,0,1">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>课程评论-</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_up">
       <property name="text">
        <string>👍</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_down">
       <property name="text">
        <string>👎</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_write_comment">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>撰写评论</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,2">
     <item>
      <widget class="QListWidget" name="lw_all_comments"/>
     </item>
     <item>
      <widget class="QTextBrowser" name="tb_comment_content"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
